packages/starters/gatsby-blog/starter/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "my-gatsby-blog",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Shouldn't we make a major instead?

Reply:
I think it doesn't matter because this is a private package, it doesn't get published on npm. But I couldn't find a way to get Kerns to ignore it

Reply:
You can use the `npx lerna publish --no-push --no-git-tag-version ` command